src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -506,13 +510,20 @@ void RadioSpinel<InterfaceType, ProcessContextType>::HandleReceivedFrame(void)
     otError        error = OT_ERROR_NONE;
     uint8_t        header;
     spinel_ssize_t unpacked;
+    spinel_iid_t   iid;
 
     LogSpinelFrame(mRxFrameBuffer.GetFrame(), mRxFrameBuffer.GetLength(), false);
     unpacked = spinel_datatype_unpack(mRxFrameBuffer.GetFrame(), mRxFrameBuffer.GetLength(), "C", &header);
 
-    VerifyOrExit(unpacked > 0 && (header & SPINEL_HEADER_FLAG) == SPINEL_HEADER_FLAG &&
-                     SPINEL_HEADER_GET_IID(header) == 0,
-                 error = OT_ERROR_PARSE);
+    // Accept spinel messages with the correct IID or broadcast IID 0.
+    iid = SPINEL_HEADER_GET_IID(header);
+    if (iid != 0 && iid != mIid)
```

Comment:
Correct, non-multipan hosts will not work with multiPAN RCP. MultiPAN RCP will reject the spinel messages from the host with iid=0 here https://github.com/openthread/openthread/blob/d15586ab515def039c93fbba09cded06d7a9f968/src/ncp/ncp_base.cpp#L343.
RCP would generate broadcast spinel frames for the received packets that have broadcast MAC dest PAN (i.e., 0xFFFF), such as beacon requests, but in general, it would depend on platform implementation.

Reply:
Yes, as you suggested, we can make the broadcast iid a configurable option.

Reply:
Made OPENTHREAD_SPINEL_CONFIG_BROADCAST_FRAME_HEADER_IID configurable option defaults to 3.